engines/ultima/ultima8/graphics/render_surface.cpp

```diff
@@ -116,16 +106,8 @@ bool RenderSurface::EndPainting() {
 	--_lockCount;
 
 	if (!_lockCount) {
-		if (_surface) {
-			// Clear pointers
-			_pixels = _pixels00 = 0;
-
-			// Render the screen if this is it (slight hack..)
-			Graphics::Screen *screen = dynamic_cast<Graphics::Screen *>(_surface);
-			if (screen)
-				screen->update();
-		}
-		// else, nothing to unlock.
+		// Clear pointers
+		_pixels = 0;
```

Comment:
minor nit, I think `nullptr` is slightly nicer than `0` here even though they have the same effect.

Reply:
Sure. Just amended the last commit to include that.